frontend/js/components/base-component.jsx

```diff
@@ -28,9 +28,8 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-
 import { connect } from 'react-redux';
-
+import _ from 'lodash';
```

Comment:
Let's start migrating away from using lodash to vanilla Javascript (Which has a much better support for functional programming and iterating objects than when we started using lodash).